mllib-tests/v1p5/src/main/scala/mllib/perf/MLAlgorithmTests.scala

```diff
@@ -488,10 +491,14 @@ abstract class DecisionTreeTests(sc: SparkContext)
 
   protected var labelType = -1
 
-  def validate(model: RandomForestModel, rdd: RDD[LabeledPoint]): Double = {
+  def validate(model: Either[RandomForestModel, GradientBoostedTreesModel], rdd: RDD[LabeledPoint])
+    : Double = {
     val numExamples = rdd.count()
     val predictions: RDD[(Double, Double)] = rdd.map { example =>
-      (model.predict(example.features), example.label)
+      model match {
```

Comment:
Can you replace this with model.map for brevity?  I haven't used Either much, so I'm not sure.